workflow/scripts/export_ariadne_variables.py

```diff
@@ -55,6 +55,47 @@ def __get_oil_fossil_fraction(n, region, kwargs):
 
     return oil_fossil_fraction
 
+# how to handle pipelines?
+# how to account for the region?
+def _get_gas_fossil_fraction(n, region, kwargs):
+    total_gas_supply =  n.statistics.supply(
+        bus_carrier="gas", **kwargs
+    ).drop("Store").groupby("carrier").sum()
+
+    drops = ["gas pipeline", "gas pipeline new"]
+    for d in drops:
+        if d in total_gas_supply.index:
+            total_gas_supply.drop(d)
+
+    gas_fossil_fraction = (
+        total_gas_supply.get("gas")
+        / total_gas_supply.sum()
+    )
+
+    return gas_fossil_fraction
+
+def _get_h2_fossil_fraction(n, region, kwargs):
+    total_h2_supply =  n.statistics.supply(
+        bus_carrier="H2", **kwargs
+    ).drop("Store").groupby("carrier").sum()
+
+    drops = ["H2 pipeline", "H2 pipeline (Kernnetz)"]
+    for d in drops:
+        if d in total_h2_supply.index:
+            total_h2_supply.drop(d)
+
+    h2_fossil_fraction_c = (
+        total_h2_supply.get(["SMR", "SMR CC"])
+        / total_h2_supply.sum()
+    )
+
+    h2_fossil_fraction = \
+        h2_fossil_fraction_c["SMR"] * n.links[n.links.carrier == "SMR"].efficiency.mean() \
+        + h2_fossil_fraction_c["SMR CC"] * n.links[n.links.carrier == "SMR CC"].efficiency.mean()
```

Comment:
h2_fossil_fraction_c is already equal to the H2 supply. So multiplying with efficiency here does not give the actual amount of H2 but less than that. 

maybe we  need two streams for H2 via SMR: h2_SMR_fraction and h2_SMRCC_fraction, since both differ in carbon intensity (which is the efficiency towards the co2 atmosphere bus)